crates/ibc/src/applications/transfer/events.rs

```diff
@@ -23,6 +23,7 @@ pub struct RecvEvent {
     pub receiver: Signer,
     pub denom: PrefixedDenom,
     pub amount: Amount,
+    pub memo: Memo,
```

Comment:
`TimeoutEvent` will also need the same `memo` field (see [ibc-go](https://github.com/cosmos/ibc-go/blob/295bab01f633633e0d6136ba9b974c0e248edbe5/modules/apps/transfer/ibc_module.go#L297))